test/e2e/azure_test.go

```diff
@@ -571,7 +572,7 @@ var _ = Describe("Workload cluster creation", func() {
 					ClusterName:              clusterName,
 					KubernetesVersion:        kubernetesVersion,
 					ControlPlaneMachineCount: pointer.Int64Ptr(1),
-					WorkerMachineCount:       pointer.Int64Ptr(1),
+					WorkerMachineCount:       replicas,
```

Comment:
nit: Is there a reason why `replicas` is stored as a variable here, but in other tests in this file, they are declared directly?

Reply:
I think this is left over from when I was using this value in another place too. I'll change this back.